tests/tape_tests/test_ensemble_frame.py

```diff
@@ -89,7 +93,7 @@ def test_frame_propagation(data_fixture, request):
 
     # Test that the output of an EnsembleFrame query is still an EnsembleFrame
     queried_rows = ens_frame.query("flux > 3.0")
-    assert isinstance(queried_rows, EnsembleFrame)
+    assert isinstance(filtered_frame, EnsembleFrame)
```

Comment:
To confirm, are you checking that filtered_frame is unmodified in these attributes?

I was expecting checks of the the of queried_rows, etc.

Reply:
Yeah this was defintely a mistake. I decided to go ahead and fix in this PR (though can break out if preferred) by wrapping _Frame.query in ensemble_frame.py

I didn't think that should be required, so in the morning will do some more checks on row-wise transformations and verify if geopandas is doing something that we're not doing.

Reply:
After taking another look, I believe the above fix should be sufficient for this PR, and I've updated the PR description to note the fix. 

I will try to substitute all uses of the source and object table for the new classes soon, and it will be interesting to see if anything else breaks.